fairlearn/metrics/__init__.py

```diff
@@ -48,13 +51,16 @@
 
 # Add the generated metrics of the form and
 # `<metric>_{difference,ratio,group_min,group_max`
-globals().update(_derived_metric_dict)
+_module_obj = sys.modules[__name__]
```

Comment:
I'm not sure this is _much_ better than `globals()` but at least it doesn't make me instantly worried.

Reply:
this looks better to me